### ISSUE TITLE
Image Augmentations for GAN Training 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#56]
# TL;DR
A study shows that FID scores were significantly improved by applying data augmentation to both Fake and Real images during GAN training. Furthermore, Consistency (BCR), a constraint that makes the discriminant values unchanged before and after data augmentation, and Contrastive loss (Cntr), which is used in self-supervised learning, were also effective.

# Why it matters:

# Paper URL
https://arxiv.org/abs/2006.02595

# Submission Dates(yyyy/mm/dd)
2020/06/04

# Authors and institutions

Zhengli Zhao, Zizhao Zhang, Ting Chen, Sameer Singh, Han Zhang
- UC Irvine 
- Google Research 

# Methods
Use Data augmentations when discriminator training.
Consistency (BCR), a constraint that makes the discriminant values unchanged before and after data augmentation, and Contrastive loss (Cntr), which is used in self-supervised learning, were also effective.
![image](https://user-images.githubusercontent.com/24524018/91297161-cb6b9900-e7d8-11ea-8ebe-c2dd996ef2fc.png)

![image](https://user-images.githubusercontent.com/24524018/91297087-a840e980-e7d8-11ea-8219-61e9f38580e3.png)

# Results
![image](https://user-images.githubusercontent.com/24524018/91297183-d45c6a80-e7d8-11ea-8c75-53efeef89c17.png)

# Comments
